# Constants
int8 TRAY = 0
int8 BIN1 = 1
int8 BIN2 = 2
int8 BIN3 = 3
int8 BIN4 = 4
int8 BIN5 = 5
int8 BIN6 = 6
int8 BIN7 = 7
int8 BIN8 = 8

# Request
int8 location
---
# Response
bool success
sensor_msgs/Image image
nist_team/Part[] parts
